### ISSUE TITLE
[work-plan] Repeating task

### ISSUE THREAD

**Issue by SamChou19815** [cornell-dti/samwise#313]
- [x] Make complete and focus in `TaskEditor` not go through save. @meganyin13 
- [x] Bring back live edit in focus view for one time task. @meganyin13 
- [x] Nuke undo feature @pihucode 
- [x] Adding subtask should not require pressing Enter @meganyin13 
- [x] New task editor design #340 @pihucode @SamChou19815 
- [x] Auto-focus tasks due today #333 @jason-tung 
  - Blocked by lack of firebase functions. Potentially blocked by #368
